src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_NUMBER = 4;
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(COLOR_NUMBER);
+        Color color = Color.values()[index];
+        return color.name();
```

Comment:
The method 'getRandomColor' correctly uses the Random instance to generate a random index and then retrieves the color using the enum's 'name()' method as recommended in the checklist.